tools/cldr/gendatefmts2.js

```diff
@@ -174,6 +174,22 @@ var hardCodeData = {
         "#{num}w": "one#{num} v|#{num} v",
         "durationShortMonths": "one#{num} ng|#{num} ng",
         "#{num}y": "one#{num} nk|#{num} nk"
+    },
+    "ps": {
+        "E0": "ﺡ",//copied data from ar/sysres.json
```

Comment:
Arabic and Farsi are the wrong languages. Same script, different language. You should take the first character of the Pashto translation for the abbreviation.

Reply:
As far as I know, Arabic changes its form depending on the position.  and not sure if it's a good idea to take the first character and even sure it's correct or not. If so should we just follow CLDR? I've found there are the same case already. uz/Arab/sysres.json

Reply:
As we discussed, I roll back my changes. Please check them again.